roles/kubernetes-apps/external_cloud_controller/openstack/tasks/openstack-credential-check.yml

```diff
@@ -23,12 +23,5 @@
   fail:
     msg: "one of external_openstack_tenant_id or external_openstack_tenant_name must be specified"
   when:
-    - external_openstack_tenant_id is not defined or not external_openstack_tenant_id
-    - external_openstack_tenant_name is not defined
-
-- name: External OpenStack Cloud Controller | check external_openstack_tenant_name value
-  fail:
-    msg: "one of external_openstack_tenant_id or external_openstack_tenant_name must be specified"
-  when:
-    - external_openstack_tenant_name is not defined or not external_openstack_tenant_name
-    - external_openstack_tenant_id is not defined
+    - (external_openstack_tenant_id is not defined or not external_openstack_tenant_id) or
```

Comment:
I keep this "not defined" check for current manner but we could remove this "not defined" check.
I don't have a strong opinion about that, it is fine to remove it if we can say that removal is safe.